pandas/core/window/expanding.py

```diff
@@ -58,6 +59,16 @@ class Expanding(_Rolling_and_Expanding):
     _attributes = ["min_periods", "center", "axis"]
 
     def __init__(self, obj, min_periods=1, center=False, axis=0, **kwargs):
+        if center:
```

Comment:
will this `center` kwarg be deprecated entirely? so users shouldn't use `center` no matter if this is set to `True` or `False`, and now using `if center` won't raise warning if setting to `False` which i _think_ do not align with this deprecation purpose.

Reply:
That's a good point, basically the warning needs to be raised whenever center is set explicitly.